### ISSUE TITLE
Jaws cant read h1s when they are nested inside fieldsets - fieldset component

### ISSUE THREAD

**Issue by Jagordon** [alphagov/govuk-frontend#1880]
Jaws cant read heading tags when they are nested inside fieldsets, as detailed here: https://github.com/FreedomScientific/VFO-standards-support/issues/100

We had this come out in an accessibility audit, using ie11 & Jaws 2019. 

I understand the html for the fieldset component is valid, and the issue is a known one with JAWs im not sure anything should be done. Just worth raising as we could change the fieldset component to avoid this issue for our users.

**Comment by hannalaakso** [comment]
@Jagordon Thanks for flagging this. I'm assuming this refers to the [legends as page headings pattern](https://design-system.service.gov.uk/get-started/labels-legends-headings/#legends-as-page-headings)? We've previously investigated this and [commented on the same Jaws issue](https://github.com/FreedomScientific/VFO-standards-support/issues/100#issuecomment-545383381). See https://github.com/alphagov/govuk-frontend/issues/1605#issuecomment-545484044: 
> Given that this is a known vendor issue and a recent regression (Jaws 17 and 18 are not impacted), we're not going to take further action now and I'm going to close the issue. If we get further feedback on this we'll reconsider the approach.

I'm going to close your issue as it's a duplicate of https://github.com/alphagov/govuk-frontend/issues/1605. 

Thanks again for bringing this to our attention 👍

**Comment by Jagordon** [comment]
Ahh apologies for raising a duplicate issue 👍

**Comment by hannalaakso** [comment]
No worries at all!
